test/src/main/java/io/strimzi/test/k8s/KubeClient.java

```diff
@@ -232,6 +232,14 @@ public List<String> listPodNamesInSpecificNamespace(String namespaceName, String
             .collect(Collectors.toList());
     }
 
+    public PersistentVolumeClaim getPvc(String namespaceName, String pvcName) {
+        return client.persistentVolumeClaims().inNamespace(namespaceName).withName(pvcName).get();
+    }
+
+    public boolean deletePvc(String namespaceName, String pvcName) {
+        return client.persistentVolumeClaims().inNamespace(namespaceName).withName(pvcName).delete();
+    }
+
     public List<PersistentVolumeClaim> listPersistentVolumeClaims(String namespaceName, String clusterName) {
         return client.persistentVolumeClaims().inNamespace(namespaceName).list().getItems().stream()
             .filter(persistentVolumeClaim -> persistentVolumeClaim.getMetadata().getName().contains(clusterName))
```

Comment:
One method has name `listPersistentVolumeClaims`, other two have `getPvc` or `deletePvc` -> I would either keep the abbreviation or long name, not both.